modules/ROOT/pages/clauses/listing-procedures.adoc

```diff
@@ -282,91 +287,106 @@ SHOW PROCEDURES EXECUTABLE BY CURRENT USER YIELD *
 ----
 
 .Result
-[role="queryresult",options="header,footer",cols="2m,2m,1m,1m"]
+[role="queryresult",options="header,footer",cols="2m,2m,1m,1m,1m"]
 |===
-| name | description | rolesExecution | rolesBoostedExecution
+| name | description | rolesExecution | rolesBoostedExecution | ...
```

Comment:
What is the purpose of this column?

Reply:
To show that there are more columns not being shown, I noticed that we did it slightly differently in the procedures and functions sections and unified them so both sections have the same format